_health-care/_js/components/personal-information/AdditionalInformationSection.jsx

```diff
@@ -44,9 +46,12 @@ class AdditionalInformationSection extends React.Component {
 
         <div className="input-section">
           <h4>Select the VA Medical Facility which will be your preferred facility</h4>
-          <State required
+          <ErrorableSelect required
+              errorMessage={isNotBlank(this.props.data.facilityState) ? undefined : 'Please select a state'}
+              label="State"
+              options={states.USA}
```

Comment:
I wonder why `USA` is getting highlighted differently? Is it because it think it's a constant?